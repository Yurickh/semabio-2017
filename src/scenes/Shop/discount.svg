<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 85.99 54.16"><defs><style>.cls-1{fill:#58c166;}</style></defs><title>discount</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><path class="cls-1" d="M1,16.39a1,1,0,0,0-1,1v32a1,1,0,0,0,1,1H12.14a1,1,0,0,0,1-1l.11-5.12c4.63.66,7.2,1.91,16.19,5.34h0c7.33,2.79,10,4.52,18.54,4.52,8.93,0,16.91-5.56,21.27-7.81h0l13.47-6.57h0a6.22,6.22,0,0,0,2.58-2.21,4.33,4.33,0,0,0,.54-3.44h0a4.62,4.62,0,0,0-5.74-3.29h0L62.36,36.3s0,0,0-.05c0-3-2-5.55-4.64-5.59h0l-16-.3L31.82,21a1,1,0,0,0-.69-.28H13.45V17.39a1,1,0,0,0-1-1Zm1,2h9.45V27.3a1,1,0,1,0,2,0V22.68H30.73l9.86,9.39a1,1,0,0,0,.67.28l16.42.31h0c1.35,0,2.68,1.49,2.68,3.59S59,39.84,57.7,39.84H38.57a1,1,0,0,0-.72.29,1,1,0,0,0,.72,1.71H57.7A4.68,4.68,0,0,0,62,38.51l18.74-5.76a2.64,2.64,0,0,1,3.22,1.89h0a2.19,2.19,0,0,1-.3,1.85A4.33,4.33,0,0,1,81.85,38h0L68.37,44.55h0C63.71,47,56.08,52.16,48,52.16c-8.28,0-10.39-1.55-17.82-4.38h0C21,44.29,18.3,43,13.29,42.28L13.45,35a1,1,0,0,0-1-1,1,1,0,0,0-.72.28,1,1,0,0,0-.31.7L11.16,48.4H2v-30Z"/><path class="cls-1" d="M59.9.3a1,1,0,0,0-.69.3L42.53,17.28a1,1,0,1,0,1.41,1.41L60.63,2A1,1,0,0,0,59.9.3Z"/><path class="cls-1" d="M45.28,0A3.53,3.53,0,0,0,43.6.47h0a3.57,3.57,0,0,0-1.3,4.84h0a3.55,3.55,0,0,0,6.14-3.55h0A3.55,3.55,0,0,0,45.28,0Zm.06,2a1.52,1.52,0,0,1,1.38.77h0A1.55,1.55,0,0,1,44,4.32,1.54,1.54,0,0,1,44.6,2.2h0A1.58,1.58,0,0,1,45.34,2Z"/><path class="cls-1" d="M57.2,12.87a3.48,3.48,0,0,0-1.84.47h0a3.55,3.55,0,0,0-1.29,4.84h0a3.56,3.56,0,0,0,4.83,1.29h0a3.56,3.56,0,0,0,1.31-4.84h0a3.57,3.57,0,0,0-3-1.77Zm-.1,2a1.55,1.55,0,0,1,.81,2.89h0a1.54,1.54,0,0,1-2.1-.57h0a1.52,1.52,0,0,1,.56-2.11h0a1.53,1.53,0,0,1,.73-.21Z"/></g></g></svg>