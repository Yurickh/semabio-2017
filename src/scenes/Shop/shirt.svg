<?xml version="1.0" encoding="UTF-8"?>
<svg width="29px" height="24px" viewBox="0 0 29 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <!-- Generator: Sketch 45.2 (43514) - http://www.bohemiancoding.com/sketch -->
    <title>shirt</title>
    <desc>Created with Sketch.</desc>
    <defs></defs>
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="BetterDesk" transform="translate(-565.000000, -3881.000000)" fill="#58C166">
            <g id="Group-3" transform="translate(520.000000, 3865.000000)">
                <g id="Group-6" transform="translate(45.000000, 13.000000)">
                    <g id="shirt" transform="translate(0.000000, 3.000000)">
                        <path d="M22.6875444,24 L6.16454964,24 C5.86900449,24 5.63021045,23.7522902 5.63021045,23.4477426 L5.62303418,13.543967 L0.471937105,10.7437033 C0.24456652,10.6198106 0.0800981357,10.402828 0.0223218421,10.1486095 C-0.0366260874,9.88576312 0.0223218421,9.61232116 0.185618591,9.39851725 L6.18937367,1.51274782 C6.21514966,1.47967444 6.27636763,1.41784159 6.30836794,1.3909742 C6.88305533,0.554603564 7.99120318,0 9.18715049,0 L10.8659582,0 C11.1222536,0 11.3436195,0.187996077 11.391144,0.448723326 C11.6762909,1.98606806 12.9829578,3.1026921 14.4995672,3.1026921 C16.0158837,3.1026921 17.323649,1.98727898 17.6088691,0.448723326 C17.657492,0.189207002 17.8777595,0 18.1340549,0 L19.7684869,0 C20.8548862,0 21.8325431,0.451977686 22.4787736,1.24581458 C22.6088984,1.30666356 22.7227667,1.39748292 22.8105662,1.51183963 L28.8154929,9.39851725 C28.9776912,9.61232116 29.0365659,9.88576312 28.9776912,10.1486095 C28.9218921,10.3995737 28.7574237,10.6174644 28.5280028,10.7437033 L23.2281811,13.6240394 L23.2281811,23.1476603 C23.2281811,23.1571206 23.2218836,23.2960743 23.220712,23.3058374 L23.2218836,23.4468344 C23.2230552,23.7522902 22.9833092,24 22.6875444,24 Z M6.69281096,22.89435 L22.1532052,22.89435 L22.1592099,13.290581 C22.1592099,13.0857076 22.2699295,12.8965006 22.4447229,12.8012917 L27.8200418,9.8792544 L21.9701371,2.19654193 C21.8356919,2.17323162 21.8142363,2.16460378 21.7324415,2.05138231 C21.2930781,1.44977973 20.5777211,1.10595277 19.7684869,1.10595277 L18.5516698,1.10595277 C18.016159,2.92854598 16.3776995,4.20834214 14.4995672,4.20834214 C12.6222404,4.20834214 10.9826092,2.92854598 10.4483433,1.10595277 L9.18715049,1.10595277 C8.33017208,1.10595277 7.54671382,1.49148095 7.08875065,2.13803912 C7.06099753,2.17648598 7.01318014,2.22552844 6.97605393,2.25542315 L1.17989808,9.87955713 L6.40649244,12.721522 C6.58216459,12.8175635 6.69171255,13.0056352 6.69171255,13.2107356 L6.69171255,22.8946527 L6.69281096,22.8946527 L6.69281096,22.89435 Z" id="Fill-1"></path>
                        <g id="Group-10" transform="translate(5.000000, 10.000000)">
                            <path d="M18.1014096,5.08234962 C17.5508113,5.08234962 17.1592186,4.81455283 17.1592186,4.4784874 L17.1592186,0.804197478 C17.1592186,0.0774790367 18.2174626,0.140928347 18.2174626,0.182188866 C18.2174626,0.223449384 18.1014096,5.08234962 18.1014096,5.08234962 Z" id="Fill-5"></path>
                            <path d="M1.32392687,5.00995422 C0.773328547,5.00995422 0.791656631,4.92760032 0.791656631,4.59153489 L0.729175511,0.795740913 C0.729175511,0.0690224713 1.10062767,0.0295747604 1.45885092,0.0961606035 C1.81707417,0.162746447 1.68708303,5.00995422 1.32392687,5.00995422 Z" id="Fill-5-Copy" transform="translate(1.229176, 2.541767) scale(-1, 1) translate(-1.229176, -2.541767) "></path>
                        </g>
                    </g>
                </g>
            </g>
        </g>
    </g>
</svg>