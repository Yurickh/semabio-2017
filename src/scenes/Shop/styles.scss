.page-shop {
  height: 100vh;
  width: 100vw;

  background-color: #FAFAFA;
}
