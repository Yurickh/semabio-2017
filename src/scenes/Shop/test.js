import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import Shop from '../Shop'

describe('Shop', () => {
  it('should render correctly', () => {
    const button = shallow(<Shop />)
    expect(toJSON(button)).toMatchSnapshot()
  })
})
