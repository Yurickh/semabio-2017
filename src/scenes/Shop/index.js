import React, { Component } from 'react'
import ShopCard from '../../components/ShopCard'
import Title from '../../components/Title'
import Button from '../../components/Button'

import iconShirt from './shirt.svg'

import './styles.css'

class Shop extends Component {
  render() {
    return (
      <section className="page-shop">
        <Title>Quero me inscrever</Title>
        <p className="description">
          Escolha o pacote ideal para você!
        </p>

        <main className="options">
          <ShopCard
            title="Básico"
            price={19}
            level={0}
            description={`
              Lorem ipsum dolor
              Consectetur adipiscing elit
              Maecenas ante ipsum
              Etiam luctus rhoncus urna
            `}
          />
          <ShopCard
            title="Médio"
            price={39}
            level={1}
            description={`
              Lorem ipsum dolor
              Consectetur adipiscing elit
              Maecenas ante ipsum
              Etiam luctus rhoncus urna
            `}
          />
          <ShopCard
            title="Super"
            price={99}
            level={2}
            description={`
              Lorem ipsum dolor
              Consectetur adipiscing elit
              Maecenas ante ipsum
              Etiam luctus rhoncus urna
            `}
          />
          <ShopCard
            title="Ultra"
            price={99.99}
            level={3}
            description={`
              Lorem ipsum dolor
              Consectetur adipiscing elit
              Maecenas ante ipsum
              Etiam luctus rhoncus urna
            `}
          />
        </main>

        <footer className="shirt">
          <Button color='outline'>
            <img
              className="icon"
              alt='icone de camiseta'
              src={iconShirt}
            />
            <div className="text">
              Quero apenas a camiseta
            </div>
          </Button>
        </footer>
      </section>
    )
  }
}

export default Shop
