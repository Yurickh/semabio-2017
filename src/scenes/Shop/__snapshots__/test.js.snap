// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shop should render correctly 1`] = `
<section
  className="page-shop"
>
  <_
    name="shop"
  >
    <Title>
      Quero me inscrever
    </Title>
  </_>
  <p
    className="subtitle"
  >
     Escolha o pacote ideal para você! 
  </p>
  <main
    className="options"
  >
    <ShopCard
      description={
        Array [
          "Ciclo de Palestras",
          "Coffee Break",
          "Kit Semabio",
        ]
      }
      level={0}
      price={40}
      title="Pacote A"
    />
    <ShopCard
      description={
        Array [
          "Ciclo de Palestras",
          "Coffee Break",
          "Kit Semabio",
          "1 Minicurso",
        ]
      }
      level={1}
      price={50}
      title="Pacote B"
    />
    <ShopCard
      description={
        Array [
          "Ciclo de Palestras",
          "Coffee Break",
          "Kit Semabio",
          "1 Minicurso",
          "Camiseta",
        ]
      }
      level={2}
      price={55}
      title="Pacote C"
    />
    <ShopCard
      description={
        Array [
          "Ciclo de Palestras",
          "Coffee Break",
          "Kit Semabio",
          "2 Minicursos",
          "Camiseta",
        ]
      }
      level={3}
      price={60}
      title="Pacote D"
    />
    <ShopCard
      description={
        Array [
          "Ciclo de Palestras",
          "Coffee Break",
          "Kit Semabio",
          "3 Minicursos",
          "Camiseta",
        ]
      }
      level={4}
      price={70}
      title="Pacote E"
    />
  </main>
  <footer
    className="shirt"
  >
    <Button
      color="outline"
      onClick={[Function]}
    >
      <img
        alt="icone de camiseta"
        className="icon -shirt"
        src="shirt.svg"
      />
      <div
        className="text"
      >
        Quero apenas a camiseta
      </div>
    </Button>
    <Button
      color="outline"
      onClick={[Function]}
    >
      <img
        alt="icone de desconto"
        className="icon -discount"
        src="discount.svg"
      />
      <div
        className="text"
      >
        Sou aluno da Assistência Estudantil
      </div>
    </Button>
  </footer>
  <span
    className="price"
  >
    A camiseta avulsa estará disponível por R$ 25,00
  </span>
  <Modal
    ariaHideApp={true}
    bodyOpenClassName="ReactModal__Body--open"
    className="modal"
    closeTimeoutMS={0}
    contentLabel="DDS Modal"
    isOpen={false}
    onRequestClose={[Function]}
    parentSelector={[Function]}
    portalClassName="ReactModalPortal"
    shouldCloseOnOverlayClick={true}
    shouldFocusAfterRender={true}
    style={
      Object {
        "overlay": Object {
          "zIndex": 5,
        },
      }
    }
  >
    <h1>
      Você sabia que...
    </h1>
    <p>
      Alunos participantes do programa de assistência estudantil da UnB têm direito a 50% de desconto em sua inscrição para a XVI Semana da Biologia?
    </p>
    <p>
      Basta nos enviar por e-mail a sua declaração (atualizada) de participação no programa, juntamente com um documento de identificação com foto, que iremos avaliar sua documentação e lhe responder com um código de uso único. Depois, é só realizar sua inscrição como qualquer outro participante e, no campo de códigos promocionais, inserir o código enviado para que 50% do valor total do seu pacote de inscrição seja abatido!
    </p>
    <p>
      Fácil, não é? Vem com a gente aproveitar esse evento que foi pensado também para você!
    </p>
    <div
      className="footer"
    >
      <div
        className="bar"
      />
      <p>
        O email deve ser enviado para o endereço 
        <strong>
          semabio.unb.bsb@gmail.com
        </strong>
         com o assunto “comprovante -programa de assistência”.
      </p>
      <p>
        Todos os códigos só poderão ser usados uma única vez e serão atrelados aos dados de cada participante, após o uso, o código será invalidado.
      </p>
    </div>
    <div
      className="close"
      onClick={[Function]}
    />
  </Modal>
</section>
`;
